presto-kinesis/src/main/java/io/prestosql/plugin/kinesis/KinesisConfig.java

```diff
@@ -44,6 +44,7 @@
     private Duration checkpointInterval = new Duration(60000, TimeUnit.MILLISECONDS);
     private String logicalProcessName = "process1";
     private int iteratorNumber;
+    private int updateInterval = 600;
```

Comment:
It would be better to use Duration type as `checkpointInterval`.

Reply:
done